dev-games/ogre/ogre-1.10.11.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils cmake-utils vcs-snapshot
+
+DESCRIPTION="Object-oriented Graphics Rendering Engine"
+HOMEPAGE="http://www.ogre3d.org/"
+SRC_URI="https://github.com/OGRECave/${PN}/archive/v${PV}.zip -> ${P}.zip"
+
+LICENSE="MIT public-domain"
+SLOT="0/1.10.0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
```

Comment:
does it make sense to provide an opengl flag when the ebuild hard-depends on virtual/opengl anyway?

Reply:
You are right. I went through all USE flags but missed that one. It really doesn't make any sense...